middlewares/validators/user.js

```diff
@@ -137,6 +137,7 @@ async function getUsers(req, res, next) {
         .messages({
           "string.empty": "prev value cannot be empty",
         }),
+      q: joi.string().optional(),
```

Comment:
can we change it to query or something better? The name q is not that explanatory.

Reply:
changed it to query